packages/suite/src/components/wallet/Fees/CollapsibleFees/CollapsibleFees.tsx

```diff
@@ -69,26 +78,32 @@ export function CollapsibleFees({
             <Collapsible gap={20}>
                 <Row justifyContent="space-between" gap={12}>
                     <CollapsibleFeesHeader label={label} typographyStyle={headerTypographyStyle} />
-                    <Collapsible.Toggle data-testid="@wallet/fees/collapsible-fees-toggle">
+                    <Collapsible.Toggle
+                        data-testid="@wallet/fees/collapsible-fees-toggle"
+                        disabled={!collapsible}
```

Comment:
Just a nit: we use isDisabled, not just disabled. But isn't it perhaps better to do a ternary over the whole return and return either a Collapsible wrapped Row or one without a Collapsible at all? Perhaps a little less DRY but more readable, since we have all those collapsible condition inside the tree which we could remove that way